### ISSUE TITLE
Consider more accurate observation count in `glimpse()` for SQL backends?

### ISSUE THREAD

**Issue by javierluraschi** [tidyverse/dplyr#3099]
Moving from [/sparklyr/issues/647](https://github.com/rstudio/sparklyr/issues/647): 

```
memdb_frame(a = seq_len(150)) %>% glimpse()
```

```
Observations: 25
Variables: 1
$ a <int> 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 2...
```

Notice that there are no 25 observations but rather 150, maybe:

```
Observations: ?
Variables: 1
$ a <int> 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 2...
```

is more appropiate?

**Comment by JohnMount** [comment]
This may seem like a small thing.  But when training a team new to R/spark/sparlyr to use a new infrastructure there are enough rough edges already.  Having `glimpse` say "`?`" would be a big improvement and avoid some "well, it said that- but please ignore it" conversations.

**Comment by kevinykuo** [comment]
`?` good

**Comment by JohnMount** [comment]
Sounds good, thanks.

**Comment by hadley** [comment]
Once tibble PR is merged, just need to remove `glimpse.tbl_sql` and bump the tibble version in `DESCRIPTION`

**Comment by krlmlr** [comment]
tibble PR merged, version bumped to 1.3.4.9002.
